eng/build.sh

```diff
@@ -144,6 +145,7 @@ cmakeargs=''
 extraargs=''
 crossBuild=0
 portableBuild=1
+keepDebugInfo=0
```

Comment:
Seems unused? (Maybe from an initial implementation more like `portableBuild` but it ended up being simpler?)

Reply:
Thanks, removed now.